selinux/services/dashboard.cil

```diff
@@ -0,0 +1,4 @@
+(type dashboard_t)
+(call system_service_p (dashboard_t))
+(allow init_t init_exec_t (file (execute)))
```

Comment:
We should probably make `system_service_p` and `client_p` accept the executable label as well, so that we allow init to execute that directly in a macro.